src/utils/fetchClient.ts

```diff
@@ -0,0 +1,50 @@
+import { Todo } from '../types/Todo';
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const BASE_URL = 'https://mate.academy/students-api/todos';
+
+// returns a promise resolved after a given delay
```

Comment:
Remove all comments. Try it write obvious code that explain all we need without comments)